bundles/binding/org.openhab.binding.isy/src/main/java/org/openhab/binding/isy/ISYBindingConfig.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.isy;
+
+import org.apache.commons.lang.StringUtils;
+import org.openhab.binding.isy.internal.ISYControl;
+import org.openhab.binding.isy.internal.ISYNodeType;
+import org.openhab.core.binding.BindingConfig;
+import org.openhab.core.items.Item;
+
+/**
+ * Holds binding configuration 
+ *
+ * @author Tim Diekmann
+ *
+ */
+public class ISYBindingConfig implements BindingConfig {
+	
+	/**
+	 * Constructor 
+	 * 
+	 * @param item full item from openhab
+	 * @param type the node type in ISY, rough translation.
+	 * @param controller the address of the controller.
+	 * @param address the address of the item to monitor for changes. same as controller if not specified.
+	 */
+	public ISYBindingConfig(Item item, ISYNodeType type, String controller,
+			String address) {
+		this(item,type,controller,address,ISYControl.ST);
+	}
+
+	/**
+	 * Constructor 
+	 * 
+	 * @param item full item from openhab
+	 * @param type the node type in ISY, rough translation.
+	 * @param controller the address of the controller.
+	 * @param address the address of the item to monitor for changes. same as controller if not specified.
+	 * @param command the command the system is sending for this controller.
+	 */
+	public ISYBindingConfig(Item item, ISYNodeType type, String controller,
+			String address, ISYControl command) {
+		this.item = item;
+		this.type = type;
+		this.controller = controller;
+		if(StringUtils.isNotBlank(address)) {
+			this.address = address;
+		} else {
+			this.address = controller;
+		}
+		if(command != null) {
+			this.controlCommand = command;
+		} else {
+			this.controlCommand = ISYControl.ST;
+		}
+	}
+	
+	private final Item item;
```

Comment:
Bindings should not hold references to Item objects since they have a different lifecycle, but since the binding needs the acceptedDataTypes and acceptedCommandTypes, it's OK to copy and the item's lists of State and Command classes.